proposed/clock-meta.md

```diff
@@ -41,17 +39,37 @@ calling `time()` or `date()`.
   retrieving the current time. This is left up to the implementation.
+* This PSR does not handle any Scheduler specific tasks like `sleep` or `wait` as those – even though they take a time or an interval as parameter – are not related to retrieving the current time. Handling such tasks is left to the respective implementations or a different PSR.
 
 ## 4. Approaches
 
 ### 4.1 Chosen Approach
 
-We have decided to formalize the existing practices, used by several other packages
-out in the wild. Some of the popular packages providing this functionality are: 
-`lcobucci/clock`, `kreait/clock`, `ergebnis/clock`, and `mangoweb/clock`. Some providing
+We have decided to formalize the existing practices, used by several other packages out in the wild. Some of the popular packages providing this functionality are: 
+
+* [`lcobucci/clock`](https://packagist.org/packages/lcobucci/clock)
+* [`kreait/clock`](https://packagist.org/packages/kreait/clock)
+* [`ergebnis/clock`](https://packagist.org/packages/ergebnis/clock)
+* [`mangoweb/clock`](https://packagist.org/packages/mangoweb/clock)
+
+(This list is not exhaustive!)
+
+Some providing
 interfaces, and some relying on overloading (extending) the Clock class to mock the
 current time.
 
+These implementations all provide a `now`-method which returns a `DateTimeImmutable`-object. As the `DateTimeImmutable`-object allows to retieve the Unix-Timestamp by calling `getTimestamp()` or `format('u.U')` this interface does not define any special methods to retieve a Unix-Timestamp or any other form of time-information apart from retrieving the `DateTimeImmutable`-object. 
+
+### 4.2 Timezones
+
+Time by now is defined by interaction of electromagnetic radiation with the excited states of certain atoms where the SI defines one second as the duration of 9192631770 cycles of radiation corresponding to the transition between two energy levels of the ground state of the caesium-133 atom at 0K. This means that retrieving the current time will always return the same time, no matter where it is observed. While the timezone defines *where* the time was observed it does not modify the actual "slice" of time.
+
+This means that for the sake of this PSR the timezone is irrelevant as it is an implementation detail of the code implementing the interface. 
+
+As there is no way to assert a certain timezone in the returnd `DateTimeImmutable` it is up to the implementation to make sure that the timezone is handled according to the Business-Logic. That is either by making sure that a call to `now()` will only return a `DateTimeImmutable`-object with a known timezone (implicit contract) or by making sure that the Business-logic handles the timezone explicitly. This an be done by calling `setTimezone` or by using `getTimestamp` which will automatically return the UTC based Unix-Timestamp.
+
+These are though actions that are not part of the contract which this interface defines.
```

Comment:
```suggestion
```

Reply:
We should make it explicit that handling the timezones are not part of the interfaces contract!